### ISSUE TITLE
`WindowInsets.isImeVisible` is temporary true during snapshot test.

### ISSUE THREAD

**Issue by nvkleban** [cashapp/paparazzi#1293]
**Description**
I have a compose UI function which hides some of it's content in case if  `ime` is visible. Inside it I have `AnimatedVisibility`.

```
AnimatedVisibility(visible = !WindowInsets.isImeVisible) {
        content()
}
```

`Paparazzi.snapshot` for screen with this component produces screenshot without `content()`

For testing purposes:
- I've added a second call to `Paparazzi.snapshot` right after the first one - and it produces correct picture with visible content.
- I've also tried a version of `Paparazzi.snapshot` with offset without any luck.

What I see from logs inside my composable function:
```
isImeVisible = true
isImeVisible = true
isImeVisible = false
isImeVisible = false
```

Maybe first values affects AnimatedVisibility somehow, idk.

**Steps to reproduce**
[PaparazziImeReproducer.zip](https://github.com/cashapp/paparazzi/files/14296454/PaparazziImeReproducer.zip)

**Expected behavior**
snapshot produces picture not affected by initial keyboard state

**Additional information:**
- Paparazzi Version: 1.3.2
- OS: Mac OS
- Compile SDK: 34
- Gradle Version: 8.4
- Android Gradle Plugin Version: 8.2.2

**Comment by jrodbx** [comment]
@nvkleban mind providing a repro project?

**Comment by nvkleban** [comment]
@jrodbx updated description with repro project

**Comment by geoff-powell** [comment]
Looking at your project, layout lib behaves the exact same way.
### Initial Preview in Android Studio
![Screenshot 2024-03-18 at 1 15 57 PM](https://github.com/cashapp/paparazzi/assets/18534850/960c0454-3e5a-4292-a82a-934efc28be11)

### Preview when running interactive mode
![Screenshot 2024-03-18 at 1 16 42 PM](https://github.com/cashapp/paparazzi/assets/18534850/04c6478f-bf90-4433-9a0f-975a1e389288)

So it seems like this is working as expected in term of how layout lib works here.

**Comment by jrodbx** [comment]
@nvkleban i'm looking at the sample project and I don't see any log statements.  How are you setting up your logging?

If I just past the following into MainActivity:

```
        val imeVisible = WindowInsets.isImeVisible
        Log.d("JROD", "isImeVisible = $imeVisible")
        AnimatedVisibility(visible = !imeVisible) {
            Text(
                text = "Visible with closed keyboard",
                modifier = modifier
            )
        }
```

I see the following in Logcat, which appears normal:
```
2024-05-29 20:18:52.334 19170-19170 JROD                    com.example.reproducer               D  isImeVisible = true
```

What am I missing?

**Comment by jrodbx** [comment]
@geoff-powell shared that testing preview appears to behave the same way.

I took a look as well via the debugger and saw two logs (not sure what you were seeing four?) confirm the true, then false values.  Going on a hunch, I searched on [Google's issue tracker](https://issuetracker.google.com/issues?q=isImeVisible) which seems to confirm that this is a problem with Compose's handling of the IME.  [This comment](https://issuetracker.google.com/issues/273300325) in particular stands out.

Closing as an issue with Compose libraries, and not Paparazzi.
